ext/sqlite3/sqlite3.c

```diff
@@ -2347,6 +2360,9 @@ static void sqlite3_param_dtor(zval *data) /* {{{ */
 /* {{{ PHP_MINIT_FUNCTION */
 PHP_MINIT_FUNCTION(sqlite3)
 {
+	/* Register SQLite3Exception class */
+	php_sqlite3_exception_ce = register_class_SQLite3Exception(spl_ce_RuntimeException);
```

Comment:
I don't think this exception should be extending RuntimeException, it should be it's own dedicated Exception.

Especially as SPL is crap.

Reply:
OK I thought it was the best course as PDO used this, so it made sense to have the same behaviour as PDO. I changed to \Exception instead.

Reply:
No worries. :)